erpnext/vehicles/doctype/vehicle_booking_order/vehicle_booking_order.py

```diff
@@ -729,6 +736,13 @@ def send_notification_on_payment_due(self):
 	def send_notification_on_cancellation(self):
 		enqueue_template_sms(self, notification_type="Booking Cancellation")
 
+	def send_customer_vehicle_anniversary_notification(self):
+		args = {
+			'customer_name': self.transfer_customer_name if self.transfer_customer else self.customer_name
+		}
+		context = {'args': args}
+		enqueue_template_sms(self, notification_type="Vehicle Anniversary", context=context, allow_if_already_sent=1)
```

Comment:
we can remove `allow_if_already_sent=1` since the framework will check notification count for that row instead of the whole document